src/Microsoft.AspNetCore.Mvc.Core/DefaultApiConventions.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Http;
+
+namespace Microsoft.AspNetCore.Mvc
```

Comment:
Did you put this in the main namespace because you expect user code to refer to it frequently?

Reply:
Right. I'd expect this to be used with the same level of frequency as `ProducesResponseType` in particular when you're migrating code.